test/intl402/RelativeTimeFormat/prototype/formatToParts/unit-invalid.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.RelativeTimeFormat.prototype.formatToParts
+description: Checks the handling of invalid unit arguments to Intl.RelativeTimeFormat.prototype.formatToParts().
+info: |
+    SingularRelativeTimeUnit ( unit )
+
+    10. If unit is not one of "second", "minute", "hour", "day", "week", "month", "quarter", "year", throw a RangeError exception.
+
+features: [Intl.RelativeTimeFormat]
+---*/
+
+const rtf = new Intl.RelativeTimeFormat("en-US");
+
+assert.sameValue(typeof rtf.formatToParts, "function");
```

Comment:
Same as above, but different existing file: [prop-desc.js](https://github.com/tc39/test262/blob/master/test/intl402/RelativeTimeFormat/prototype/formatToParts/prop-desc.js)

Reply:
I added these because I kept forgetting that SpiderMonkey doesn't implement formatToParts, so I was wasting time trying to figure out why calling undefined threw a TypeError. I can remove if you prefer.